torchao/float8/config.py

```diff
@@ -234,6 +248,9 @@ class Float8LinearConfig:
     # tests so that the warning does not spam the CI stdout.
     force_recompute_fp8_weight_in_bwd: bool = False
 
+    # configuration used for calculating the scaling factor used in float8 quantization.
+    scaling_factor_config: Float8ScalingFactorConfig = None
```

Comment:
just make it a boolean, and set it to true for rowwise scaling in code using `Float8LinearRecipeName`?

Reply:
> there didn't seem to be a nice/simple way for the user to use row-wise scaling without power of 2 scale factors

they could create a Float8LinearConfig and configure whatever they want?

Reply:
> Default on for row-wise scaling, with option to disable. For other scaling granularities, default off, with option to enable.

that should be changed to default on for row-wise scaling recipe created by the recipe enum to recipe function

Reply:
> > Default on for row-wise scaling, with option to disable. For other scaling granularities, default off, with option to enable.
> 
> that should be changed to default on for row-wise scaling recipe created by the recipe enum to recipe function

Yeah, this is what I did in the other branch. I can create a PR for that one as well to compare. I didn't prefer it because a lot of the rowwise scaling tests do not use the recipe helper function, and just directly call `hp_tensor_to_float8_dynamic` so this approach would require modifying a bunch of tests, adding new parameterizations, etc. So this approach seemed like a simpler way to me. Happy to do it the other way, though.